Shared/Samples/Change viewpoint/README.md

```diff
@@ -0,0 +1,47 @@
+# Change viewpoint
+
+Set the map view to a new viewpoint.
+
+![Image of change viewpoint 1](change-viewpoint-1.png)
+![Image of change viewpoint 2](change-viewpoint-2.png)
+
+## Use case
+
+Programmatically navigate to a specified location in the map or scene. Use this to focus on a particular point or area of interest.
+
+## How to use the sample
+
+The map view has several methods for setting its current viewpoint. Select a viewpoint from the UI to see the viewpoint changed using that method.
```

Comment:
```suggestion
The map view has several methods for setting its current viewpoint. Select a viewpoint from the toolbar and observe the viewpoint changing according to the selected method.
```
Just a suggestion, feel free to adjust how you see fit.

Reply:
I'm fine with keeping consistent with the design README.